dcm4chee-arc-war/src/main/webxml/web-secure.xml

```diff
@@ -56,6 +56,36 @@
       <role-name>${auth-user-role:user}</role-name>
     </auth-constraint>
   </security-constraint>
+
+  <security-constraint>
+    <!--
+     Only admin users should be allowed access to CTRL-RS
+       (Start/Stop/Reload Archive)
+    -->
+    <web-resource-collection>
+      <web-resource-name>dcm4chee-arc-rs-ctrl</web-resource-name>
+      <url-pattern>/ctrl/*</url-pattern>
+      <http-method-omission>OPTIONS</http-method-omission>
+    </web-resource-collection>
+    <auth-constraint>
+      <role-name>${super-user-role:admin}</role-name>
+    </auth-constraint>
+  </security-constraint>
+
+  <security-constraint>
+    <!--
+     Users in reload group should be allowed reload the archive
+    -->
+    <web-resource-collection>
```

Comment:
Added constraints for accessing `/ctrl/*` (restricted to `admin` role) and `/ctrl/reload` (restricted to a `reload role`). The latter may be too fine-grained, but it allows us to have a user in Study Management that's allowed to reload the arc config over the RESTful API, and nothing else.